modules/ROOT/pages/external-organization-access.adoc

```diff
@@ -0,0 +1,39 @@
+= External Organization Access
```

Comment:
This title seems a little nonintuitive, despite the fact that the feature we are documenting here is called External Access. 
I wonder if we can use something a bit more "friendly," like "Asset-Sharing Between Organizations" or "Cross-Organizational Collaboration and Asset Sharing" or something really wonderful that I can't find the exact words for just now...?

Reply:
This is what we decided on with legal for now :/

Reply:
Worth noting too that this feature is laying ground work for other features and is going to exist independently of the others. Keeping this puppy very isolated.